modules/zarr/test/data/ome.zarr/.zattrs

```diff
@@ -0,0 +1,65 @@
+{
+    "multiscales": [
+        {
+            "datasets": [
+                {
+                    "path": "0"
+                },
+                {
+                    "path": "1"
+                }
+            ],
+            "version": "0.1"
+        }
+    ],
+    "omero": {
```

Comment:
OME metadata in root attrs. Used to distinguish multiscale dataset _is_ OME-Zarr.